docs/tracker_stores.rst

```diff
@@ -0,0 +1,103 @@
+.. _tracker_store:
+
+
+Tracker Stores
+==============
+
+All conversations are stored within a `tracker store`.
+Rasa Core provides implementations for different store types out of the box.
+If you want to use another store, you can also build a custom tracker store by extending the `TrackerStore` class.
+
+.. contents::
+
+InMemoryTrackerStore (default)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+:Description:
+    `InMemoryTrackerStore` is the default tracker store. It is used if no other tracker store is configured.
+    It stores the conversation history in memory.
+
+    .. note:: As this store keeps all history in memory the entire history is lost if you restart Rasa Core.
+
+:Configuration:
+    To use the `InMemoryTrackerStore` no configuration is needed.
+
+
+RedisTrackerStore
+~~~~~~~~~~~~~~~~~~
+
+:Description:
+    `RedisTrackerStore` can be used to store the conversation history in `Redis <https://redis.io/>`_.
+    Redis is a fast in-memory key-value store which can optionally also persist data.
+
+:Configuration:
+    To set up Rasa Core with Redis the following steps are required:
+
+    1. Start your Redis instance
+    2. Add required configuration to your `endpoints.yml`
+
+        .. code-block:: yaml
+
+            tracker_store:
+                store_type: redis
+                url: <host of the redis instanc, e.g. localhoste>
+                port: <port of your redis instance, usually 6379>
+                db: <number of your database within redis, e.g. 0>
+                password: <password used for authentication>
+
+    3. To start the Rasa Core server using your configured Redis instance,
+       add the :code:`--endpoints` flag, e.g.:
+
+        .. code-block:: bash
+
+            python -m rasa_core.run --core models/dialogue --endpoints endpoints.yml
+
+MongoTrackerStore
+~~~~~~~~~~~~~~~~~
+
+:Description:
+    `MongoTrackerStore` can be used to store the conversation history in `Mongo <https://www.mongodb.com/>`_.
+    MongoDB is a free and open-source cross-platform document-oriented NoSQL database.
+
+:Configuration:
+    1. Start your MongoDB instance.
+    2. Add required configuration to your `endpoints.yml`
+
+        .. code-block:: yaml
+
+            tracker_store:
+                store_type: mongod
+                url: <url to your mongo instance, e.g. mongodb://localhost:27017>
+                db: <name of the db within your mongo instance, e.g. rasa>
+                user: <username used for authentication>
+                password: <password used for authentication>
+
+    3. To start the Rasa Core server using your configured Redis instance,
```

Comment:
This should probably mean MongoDB instance instead of Redis instance?

Reply:
Thanks! I updated the branch. Somehow it's not yet reflected in Github.